### PR TITLE
The Chef can now deliver food to prisoners easily!

### PR DESCRIPTION
**Pull request by Iamgoofball**
All he has to do is drop food down the prison chute, and it'll be safely converted into a friendly healthy edible substitute that is 100% safe for prisoner consumption.

_Warning: Do not put carbon or silicon based lifeforms down the chute._

### CODE REVIEW

File: code/modules/recycling/disposal/loafer.dm [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+			else
+				newLoaf.loaf_factor += (newLoaf.loaf_factor / 10) + 50
+			poorSoul.emote("scream")
+			poorSoul.die()
+		if(istype(A, /obj/item))
+			var/obj/item/I = A
+			newLoaf.loaf_factor += I.w_class * 5
+		else
+			newLoaf.loaf_factor++
+		qdel(A)
```
Comment (JJRcop) [inline code/modules/recycling/disposal/loafer.dm]: Feed changeling to prisoners! Or your target!

Comment (Iamgoofball) [reply code/modules/recycling/disposal/loafer.dm]: the chef has access to a gibber, so

**Comment by Qustinnus** [comment]
goof be a good boy and put your sprites in the goonie folder <3

**Comment by Qustinnus** [comment]
also feature looks cool 👍

**Comment by Mey-Ha-Zah** [comment]
220 Force Loaf... Goof... are you just trying to make the chef the most robust person possible?

**Review by ninjanomnom — CHANGES_REQUESTED** [review]
Put your goon sprite in the goon folder

**Comment by ninjanomnom** [comment]
I wish you'd get new sprites or just make coder sprites so we can actually make progress removing the goon folder >:(

**Comment by Iamgoofball** [comment]
Why replace what's perfect

File: code/modules/recycling/disposal/loafer.dm [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	throwforce = 0
+	volume = 1000
+	list_reagents = list("nutriment" = 3, "flour" = 2, "corn_starch" = 10, "mayonnaise" = 10, "mushroomhallucinogen" = 3, "synthflesh" = 10)
+	var/loaf_factor = 1
+	var/loaf_recursion = 1
+
+/obj/item/reagent_containers/food/snacks/prison_loaf/proc/loafing_it_up()
+	var/orderOfLoafitude = max( 0, min( round( log(8, loaf_factor)), 10 ) )
+	w_class = min(orderOfLoafitude+1, 4)
+	switch(orderOfLoafitude)
```
Comment (optimumtact) [inline code/modules/recycling/disposal/loafer.dm]: I'll need you to graph the required amount for each density

Comment (Iamgoofball) [reply code/modules/recycling/disposal/loafer.dm]: That's spoiling the fun of loafistry

**Comment by KorPhaeron** [comment]
Freeze

**Comment by evsey9** [comment]
merge NOW

File: code/modules/recycling/disposal/loafer.dm (hunk 2 above)
Comment (Iamgoofball) [reply code/modules/recycling/disposal/loafer.dm]: prelim math, 8000~ loaf factor for like, 4.8 loafitude

File: code/modules/recycling/disposal/loafer.dm [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+			throwforce = 110
+			throw_range = 1
+			reagents.add_reagent("bluespace",50)
+
+		if(10)
+			name = "strangelet loaf"
+			desc = "You should probably call a priest."
+			icon_state = "ploaf7"
+			force = 220
+			throwforce = 220
```
Comment (PKPenguin321) [inline code/modules/recycling/disposal/loafer.dm]: We're reaching levels of loaf that shouldn't even be possible

**Comment by PKPenguin321** [comment]
I love goofkitchen honestly

**Comment by PKPenguin321** [comment]
Also those new disposal pipe sprites should go in a separate PR since I think sprites bypass the freeze and they're good

**Comment by Iamgoofball** [comment]
What

**Comment by JJRcop** [comment]
@PKPenguin321 This isn't a new disposal chute subtype, it's a new pipe, those pipes.

**Comment by PKPenguin321** [comment]
Oh I see
Well those sprites should replace our current 3-directional disposal pipe sprites, the arrows are good
unless we already did that and I don't remember

**Pull request closed without merging**